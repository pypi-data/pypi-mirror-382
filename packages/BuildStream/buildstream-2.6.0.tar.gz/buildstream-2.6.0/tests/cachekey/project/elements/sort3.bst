kind: stack

depends:
- elements/sort2.bst
