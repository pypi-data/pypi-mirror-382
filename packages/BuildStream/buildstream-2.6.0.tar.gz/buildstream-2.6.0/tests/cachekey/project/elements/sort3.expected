d4fb3897bd712221b8348e6beb7a51cce381906bce7dfd05dc310d2ca8b0ed8f
